components/server/src/bitbucket-server/bitbucket-server-context-parser.ts

```diff
@@ -88,7 +88,11 @@ export class BitbucketServerContextParser extends AbstractContextParser implemen
             return await this.handleNavigatorContext(ctx, user, repoKind, host, owner, repoName, more);
         } catch (e) {
             span.addTags({ contextUrl }).log({ error: e });
```

Comment:
span.log is using the original e as well

Reply:
That's expected, since I'm happy to preserve the details in the log span, just don't want to directly log it in the console for tests.